client/src/components/settings/toggle-radio-setting.tsx

```diff
@@ -50,17 +50,24 @@ export default function ToggleRadioSetting({
               name={flagName}
               value='1'
             />
+            <span className='custom-circle'>
+              {flag ? <span className='selected-inner-circle' /> : null}
+            </span>
             <span>{restProps.onLabel}</span>
           </label>
-          <label htmlFor={secondRadioId} data-checked={!flag}>
+
+          <label htmlFor={secondRadioId}>
             <input
               id={secondRadioId}
-              type='radio'
               {...(!flag && { defaultChecked: true })}
               {...(flag && { onChange: toggleFlag })}
               name={flagName}
+              type='radio'
```

Comment:
Same as the previous comment, undoing this, will clean the changing log in  https://github.com/freeCodeCamp/freeCodeCamp/pull/50094/files

Reply:
![image](https://user-images.githubusercontent.com/103327712/233772878-72f9944f-807a-41d1-af73-d467bc0e2b9f.png)

same here, i changed the position. May I remove this change, if you would be making a commit for it?